### DISCUSSION TITLE
Poor documentation

### DISCUSSION THREAD

**Discussion by 0LostConnection** [aoijs/aoi.js#365] [General]
The documentation on the official website is confusing and out of order plus the example on the README.md isn't compatible with the last version (5.1.2).

I tried to use it but I couldn't

**Comment by Leref** [comment]
Already well known, in the upcoming v6.0.0 version,
The documentation is being auto-generated as well more specific with examples and updated constantly.

**Comment by devmike123** [comment]
> The documentation on the official website is confusing and out of order plus the example on the README.md isn't compatible with the last version (5.1.2).
> 
> I tried to use it but I couldn't

The documentation helpers are working on it and getting it fixed. Sorry for the inconvenience faced!

**Comment by Neodevils** [comment]
Documentation updates almost done.

**Comment by Leref** [comment]
Documentation is updated continuously.

**Reply by Neodevils** [reply]
That is good to know! 🙌🏻
